app/src/main/java/com/einmalfel/podlisten/PlayerService.java

```diff
@@ -110,6 +110,7 @@ public void run() {
             ct = CallbackType.PROGRESS;
           }
         } catch (InterruptedException ignored) {
+          Thread.currentThread().interrupt();
```

Comment:
doesn't seem to make sense since the thread will exit when Interrupted exception rises